components/raftstore-v2/src/tablet.rs

```diff
@@ -76,7 +76,14 @@ mod tests {
         // Setting tablet will refresh cache immediately.
         cached_tablet.set(2);
         assert_eq!(cached_tablet.cache().cloned(), Some(2));
-        assert_eq!(cached_tablet.latest().cloned(), Some(2));
+        // Test `latest()` will use cache.
+        {
```

Comment:
This is not the TiKV style.

Reply:
What is not? Be a little more specific?

Reply:
Wrapping test steps in a pair of brackets. We usually use a blank line.

Reply:
Okay. The intention really is to scope the unsafe modifications of `data`.